lib/open_pantry/web/views/shared_view.ex

```diff
@@ -11,7 +11,11 @@ defmodule OpenPantry.Web.SharedView do
     Renders a distribution line item for food selection index, or to send update to same page via channel
   """
   def render_distribution(%StockDistribution{id: id}) do
-    render_to_string(__MODULE__, "distribution.html", distribution: StockDistribution.find(id, [:stock]))
+    try do
+      render_to_string(__MODULE__, "distribution.html", distribution: StockDistribution.find(id, [:stock]))
+    rescue
```

Comment:
Awesome... I haven't done much rescuing in Elixir, but this seems like a good use case for it :-)  I imagine you hit this by necessity after other parts of fix were done?  I'd like to improve our test coverage, I had some better integration tests before the CSS redesign, and had trouble getting them to work afterwards.  There's also an open PR to move wallaby to use headless chrome that might help in fixing up integration tests if you have any interest in helping to take over that, but I think though it mostly works locally there's some work to be done getting it to work on Travis or CircleCI (I think we run both at the moment, but Circle has some issues and Travis I got working faster)

Reply:
Yes, added this by necessity because deleted StockDistributions could not be rendered for the channel's response.
In the background, "StockDistribution.find" uses "Repo.one!" which raises exceptions instead of returning tuples, so rescuing seemed like a good match.